static/app/views/settings/project/projectOwnership/index.tsx

```diff
@@ -121,12 +121,13 @@ export default function ProjectOwnership({project}: {project: Project}) {
               <Access access={['org:integrations']} project={project}>
                 {({hasAccess}) => (
                   <Button
-                    onClick={handleAddCodeOwner}
                     size="sm"
+                    onClick={handleAddCodeOwner}
                     data-test-id="add-codeowner-button"
                     disabled={!hasAccess}
+                    icon={<IconGithub />}
```

Comment:
i think we can use GitLab for codeowners too

Reply:
@roggenkemper you are correct https://docs.sentry.io/product/issues/ownership-rules/, let me remove the github icon and see if folks are ok w the change